include/pokemon_menu.h

```diff
@@ -1,6 +1,8 @@
 #ifndef GUARD_POKEMON_MENU_H
 #define GUARD_POKEMON_MENU_H
 
+#include "menu.h"
```

Comment:
dont do this. just include menu.h in the file.

Reply:
But it needs to include this so it knows how to resolve `MenuAction2 `